core/server/api/themes.js

```diff
@@ -52,7 +52,8 @@ themes = {
                     if (error.failures) {
                         _.each(error.failures, function (childError) {
                             validationErrors.push(new errors.ValidationError(i18n.t('errors.api.themes.invalidTheme', {
-                                reason: childError.ref
+                                // @TODO use property, or some other mechanism to send & display multiple values here
+                                reason: childError.message + ' (' + childError.ref + ')'
```

Comment:
I've tested with gscan `0.0.10`.

When i upload for example this theme https://www.ghostforbeginners.com/brage/, i get `Theme is invalid: undefined (/assets/css/screen.css)`.

Another issue i've mentioned was: ember only shows one error, which might not be helpful for the uploader if there are more specific errors in `failures` gscan response.

see - this is what the server right now returns, because i thought ember will show all errors:
```
{"errors":[
{"message":"Theme is invalid: undefined (/assets/css/screen.css)","errorType":"ValidationError"},
{"message":"Theme is invalid: undefined (/assets/css/icons.css)","errorType":"ValidationError"},
{"message":"Theme is invalid: undefined (/assets/fonts/fonts.css)","errorType":"ValidationError"}
,{"message":"Theme is invalid: undefined (/assets/js/main.js)","errorType":"ValidationError"},
{"message":"Theme is invalid: Assets such as CSS & JS must use the <code>{{asset}}</code> helper","errorType":"ValidationError"}]}
```

This `{"message":"Theme is invalid: Assets such as CSS & JS must use the <code>{{asset}}</code> helper","errorType":"ValidationError"}]}` is in my opinion the most helpful message for the uploader, but i see the first one in ember.